### PR TITLE
Test module reference

### PR DESCRIPTION
**Pull request by jamie-pate**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -25,7 +24,7 @@
     "lint-fix-code": "node node_modules/ftnt-devops-ci/bin/ftnt-devops-ci.js f -fl  -F .eslintignore -L .eslintignore \"**/*{.js,.json,.ts}\"",
     "lint-fix-template": "node node_modules/ftnt-devops-ci/bin/ftnt-devops-ci.js f -f -F .eslintignore -L .eslintignore \"**/*.json\" --parser \"json\"",
     "lint-fix": "npm run lint-fix-code",
-    "build-core": "cd node_modules/autoscale-core && npm install && npm run build && cd ../../",
+    "build-core": "cd node_modules/autoscale-core && npm install && npm run build && npm prune --production && cd ../../",
```
Comment (jamie-pate) [inline package.json:27]: don't need this at all

File: package.json [hunk 2]
```diff
@@ -49,8 +48,8 @@
   "homepage": "https://github.com/fortinet/fortigate-autoscale-azure#readme",
   "dependencies": {
     "@azure/functions": "^1.2.3",
+    "@fortinet/fortigate-autoscale": "https://github.com/fortinet/autoscale-core/releases/download/3.2.0/fortinet-fortigate-autoscale-3.2.0.tgz",
```
Comment (jamie-pate) [inline package.json:51]: Use npm link during dev and have a script that updates the version number when you publish a new version?

File: package.json [hunk 3]
```diff
@@ -8,7 +8,6 @@
     "doc": "docs"
   },
   "scripts": {
-    "postinstall": "npm run build-core",
```
Comment (jamie-pate) [inline package.json:11]: :nauseated_face:

File: package.json (hunk 2 above)
Comment (jamie-pate) [reply package.json:51]: at some point we should stop goofing around with this and actually set up the npm registry at github?

File: byol-license/func.ts [hunk 4]
```diff
@@ -7,7 +7,7 @@ import {
     AzureFunctionResponse,
     AzurePlatformAdaptee,
     AzurePlatformAdapter
-} from '@fortinet/fortigate-autoscale/azure';
+} from '@fortinet/autoscale/fortigate-autoscale/azure';
```
Comment (jamie-pate) [inline byol-license/func.ts:10]: this is incorrect, should be the previous

File: package.json (hunk 2 above)
Comment (JaydenLiang) [reply package.json]: there is one and will change it to update this line in the next change.

File: byol-license/func.ts (hunk 4 above)
Comment (JaydenLiang) [reply byol-license/func.ts]: fixed

**Review by JaydenLiang — APPROVED** [review]

**Pull request merged**